### PR TITLE
Added new page for maps and posters

### PR DESCRIPTION
**Pull request by Jlaird**
Screen Shot 2019-06-04 at 2 36 35 PM

### CODE REVIEW

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import getAdUnit from '@endeavorb2b/base-website-common/utils/gam/get-adunit';
+import hierarchyAliases from '@endeavorb2b/base-website-common/utils/website-section/hierarchy-aliases';
+
+$ const { site } = out.global;
+$ const section = getAsObject(data, 'section');
+$ const aliases = hierarchyAliases(section);
+$ const adSlots = {
+  'gpt-ad-lb1': getAdUnit(site, { name: 'lb1', aliases }),
+  'gpt-ad-rail1': getAdUnit(site, { name: 'rail1', aliases, size: [300, 250] }),
+  'gpt-ad-rail2': getAdUnit(site, { name: 'rail2', aliases, size: [300, 600] }),
```
[the remark this answers is not in the document]
Comment (solocommand) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: Additionally, if you don't need to include multiple units for a single request, just use the alternate form -- don't include the getAdUnit calls, don't send `slots` to `endeavor-ad-gam-head`, and call `endeavor-gam-ad-unit-define-display name="lb1"` below instead of `-display`.

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { getAsObject, getAsArray, get } from '@base-cms/object-path';
```
Comment (zarathustra323) [inline sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: `getAsArray` and `get` are not used. Remove them from the import.

File: sites/offshore-mag/server/routes/website-section.js [hunk 3]
```diff
@@ -1,8 +1,14 @@
 const queryFragment = require('@endeavorb2b/base-website-themes/pennwell/api/fragments/website-section-page');
 const { withWebsiteSection } = require('@base-cms/marko-web/middleware');
 const section = require('../templates/website-section');
+const mapspostersTemplate = require('../templates/website-section/resources-maps-posters');
 
 module.exports = (app) => {
+  app.get('/:alias(resources/maps-posters)', withWebsiteSection({
+    template: mapspostersTemplate,
```
Comment (solocommand) [inline sites/offshore-mag/server/routes/website-section.js]: Proper camelcasing please, `mapsPostersTemplate`.

File: sites/offshore-mag/server/routes/website-section.js [hunk 4]
```diff
@@ -1,8 +1,14 @@
 const queryFragment = require('@endeavorb2b/base-website-themes/pennwell/api/fragments/website-section-page');
 const { withWebsiteSection } = require('@base-cms/marko-web/middleware');
 const section = require('../templates/website-section');
+const mapspostersTemplate = require('../templates/website-section/resources-maps-posters');
```
Comment (solocommand) [inline sites/offshore-mag/server/routes/website-section.js]: Preferably, this file should be `templates/website-section/resources/maps-posters` to match the alias.

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <endeavor-ad-gam-head slots=adSlots />
+  </@head>
+
+  <@above-container>
+    <endeavor-gam-ad-unit-out-of-page name="wa" aliases=aliases />
+    <endeavor-gam-ad-unit-out-of-page name="reskin" aliases=aliases />
+    <endeavor-gam-ad-unit-display id="gpt-ad-lb1" modifiers=["top-of-page"] />
+  </@above-container>
+  
+  <div class="page-wrapper default-page">
```
Comment (zarathustra323) [inline sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: The `default-page` class should be removed

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    <div class="row">
+      <div class="col align-self-start">
```
Comment (zarathustra323) [inline sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: The `align-self-start` class isn't need (or at least shouldn't be needed). Why was this added?

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+        <div class="page-wrapper__header">
+          <endeavor-breadcrumbs-published-content title=section.name />
```
Comment (zarathustra323) [inline sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: Since this is a website section page (not a published content page), you should use the website section breadcrumbs:
```marko
<endeavor-breadcrumbs-website-section section=section />
```

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko (hunk 5 above)
Comment (Jlaird) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: If I remove this the spacing is not set correctly, should I use margin bottom then?

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko (hunk 6 above)
Comment (Jlaird) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: You are correct, I was trying to figure out why the margin on the bottom of the blocks wasn't being set. Once I used default-page it started working.

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko (hunk 2 above)
Comment (Jlaird) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: Fixed

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko (hunk 6 above)
Comment (Jlaird) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: removed

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko (hunk 5 above)
Comment (zarathustra323) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: What spacing, specifically, is incorrect?

File: sites/offshore-mag/server/routes/website-section.js (hunk 3 above)
Comment (Jlaird) [reply sites/offshore-mag/server/routes/website-section.js]: Got ya, I updated that.

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jlaird) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: @solocommand  Can you take a look again and see if that is better?

File: sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko (hunk 5 above)
Comment (Jlaird) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: @zarathustra323  This is what is looks like if I remove it.
Screen Shot 2019-06-04 at 3 41 10 PM

Comment (Jlaird) [reply sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko]: Sorry that screenshot was not the best, there is no gap between so the white from both blocks are touching.
